FirebaseABTesting/Sources/Public/FIRExperimentController.h

```diff
@@ -86,6 +88,21 @@ NS_SWIFT_NAME(ExperimentController)
 /// @param payloads   List of experiment metadata.
 - (NSTimeInterval)latestExperimentStartTimestampBetweenTimestamp:(NSTimeInterval)timestamp
                                                      andPayloads:(NSArray<NSData *> *)payloads;
+
+/// Expires experiments that aren't in the list of running experiment payloads.
+/// @param origin         The originating service affected by the experiment, it is defined at
+///                       Firebase Analytics FIREventOrigins.h.
```

Comment:
Since `FIREventOrigins.h` isn't a public header, it's probably better to remove it from this and the below method.